easybuild/easyconfigs/r/RSEM/RSEM-1.2.26_eb_provided_zlib.patch

```diff
@@ -0,0 +1,162 @@
+#Use EB provided libz and use EB provided CXX and CXXFLAGS
+# January 11, 2016 by B. Hajgato (Vrije Ubiversiteit Brussel)
+--- Makefile.orig	2016-01-08 02:23:01.000000000 +0100
++++ Makefile	2016-01-18 12:02:56.406105229 +0100
+@@ -1,6 +1,6 @@
+-CC = g++
+-CFLAGS = -Wall -c -I.
+-COFLAGS = -Wall -O3 -ffast-math -c -I.
++CXX = $(CXX)
++CFLAGS = -Wall -c -I. $(CFLAGS)
```

Comment:
@boegel
Actually there is no difference between `$()` and `${}` in `GNU make`
But your point was right, I did not check the patch. Now it is corrected, and it made possible to add the `intel/2015b` version as well.